src/main/java/ch/ejpd/lgs/searchindex/client/configuration/SedexConfiguration.java

```diff
@@ -49,9 +49,17 @@ public class SedexConfiguration {
   @Getter
   private int sedexMessageClass;
 
+  @Value("${lwgs.searchindex.client.sedex.data.processor.version}")
+  @Getter
+  private String dataProcessorVersion;
+
   @Value("${lwgs.searchindex.client.sedex.create-directories:false}")
   private boolean createDirectories;
 
+  @Value("${lwgs.searchindex.client.sedex.sender.max.length:80}")
```

Comment:
is that actually a requirement? where does this restriction come from? sedex? internal db? ...?

Reply:
It was when creating a folder for land registers. It will be removed

Reply:
removed